styles/globals.css

```diff
@@ -1,3 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.parent-div li {
+  white-space: nowrap;
+  overflow: hidden;
+  animation: 500ms widthAnimation forwards
+}
+
+
+@keyframes widthAnimation {
+  0% {
+    width: 0px;
+  }
+  100% {
+    width: 100%;
+  }
+}
```

Comment:
Innostuin CSS:n animaatioista, kun näin muilla sivuilla niitä, joten päätin itsekin yrittää jotain simppeliä. Nämä lienee järkevämpi tehdä johonkin omaan CSS-tiedostoon, esim. index.css tai muu vastaava nimi? Nyt on tuossa globals.css-tiedostossa.

Reply:
Toimii näinkin, mutta nyt kun on Tailwind käytössä niin parempi tehdä Aleksin Discordissa kuvaamalla tavalla, eli laittamalla tuo widthAnimation tailwind.config.js tiedostoon.